AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.h

```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
```

Comment:
Probably not necessary. It is already imported in header fil

Reply:
Got it, removed.